src/openrct2/actions/RideSetStatus.hpp

```diff
@@ -78,6 +78,15 @@ DEFINE_GAME_ACTION(RideSetStatusAction, GAME_COMMAND_SET_RIDE_STATUS, GameAction
             return res;
         }
 
+        if (_status >= RIDE_STATUS_COUNT)
+        {
+            log_warning("Invalid ride status", uint32_t(_rideIndex));
```

Comment:
This is not correct. The string does not contain any parameters, yet rideIndex is added.
```suggestion
            log_warning("Invalid ride status %u for ride %u", uint32_t(_status), uint32_t(_rideIndex));
```